ecl/eclagent/eclagent.cpp

```diff
@@ -2312,7 +2318,7 @@ void EclAgentWorkflowMachine::noteTiming(unsigned wfid, timestamp_type startTime
     updateWorkunitStat(wu, SSTworkflow, scope, StWhenStarted, nullptr, startTime, 0);
     updateWorkunitStat(wu, SSTworkflow, scope, StTimeElapsed, nullptr, elapsedNs, 0);
 
-    const __int64 cost = calcCost(agent.queryAgentMachineCost(), elapsedNs);
+    const __int64 cost = calcCostNs(agent.queryAgentMachineCost(), elapsedNs);
```

Comment:
Having a calcCostNS seems overkill, unlikely to be practical to get that level of prevision.
Seconds are probably enough. I'd probably convert and settle on 1 cost method.